site/content/includes/quickstart-common-steps.md

```diff
@@ -190,15 +188,36 @@ curl {{% apiendpoint %}}placement | jq .
 [Read more about the bootstrapping process](/docs/operational_guide/bootstrapping_crash_recovery/).
 {{% /notice %}}
 
+### Ready a Namespace
+<!-- TODO: Why?> -->
+Once a namespace has finished bootstrapping, you must mark it as ready before receiving traffic by using the _{{% apiendpoint %}}namespace/ready_.
```

Comment:
This should also be `services/m3db/namespace/ready` instead of `namespace/ready`